NuGetProtocolSamples/Samples.cs

```diff
@@ -0,0 +1,121 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using NuGet.Common;
+using NuGet.Packaging;
+using NuGet.Protocol.Core.Types;
+using NuGet.Versioning;
+using Xunit;
+
+namespace NuGet.Protocol.Samples
+{
+    public class Samples
+    {
+        [Fact]
+        public async Task ListPackageVersions()
+        {
+            ILogger logger = NullLogger.Instance;
+            CancellationToken cancellationToken = CancellationToken.None;
```

Comment:
Note: I purposefully do NOT use `var` in these samples! These samples are for the docs website which does not have intellisense.